.github/workflows/triage.yml

```diff
@@ -0,0 +1,53 @@
+name: Triage new issues
+
+on:
+  issues:
+    types: [ opened ]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
+  ORGANIZATION: ethereum
+  PROJECT_NUMBER: 43
+  DRY_RUN: true
+
+jobs:
+  triage_issues:
+    runs-on: ubuntu-latest
+    if: join(github.event.issue.labels) == ''
+    steps:
+      - name: Retrieve project data
+        env:
+          ORGANIZATION: ${{ env.ORGANIZATION }}
+          PROJECT_NUMBER: ${{ env.PROJECT_NUMBER }}
```

Comment:
Aren't these already in the env? Do you have to repeat them?

Reply:
Nope, they can be removed.